less/index.less

```diff
@@ -49,11 +49,226 @@ table {
 }
 
 /* Set every element's box-sizing to border-box */
+@ebony-color: #26303C;
+@light-grey: lightgrey;
```

Comment:
Nice idea -- setting colors to variable. This will come in handy for color scheming later down the line. For instance, you could have primary, secondary, tertiary variables. That way, if you ever wanted to change your color scheme, you only have to do it in this one place